crates/ruff_cli/src/stdin.rs

```diff
@@ -1,9 +1,17 @@
 use std::io;
-use std::io::Read;
+use std::io::{Read, Write};
 
 /// Read a string from `stdin`.
 pub(crate) fn read_from_stdin() -> Result<String, io::Error> {
     let mut buffer = String::new();
     io::stdin().lock().read_to_string(&mut buffer)?;
     Ok(buffer)
 }
+
+/// Read bytes from `stdin` and write them to `stdout`.
+pub(crate) fn parrot_stdin() -> Result<(), io::Error> {
```

Comment:
Nice name!

Reply:
Haha I wasn't sure if it would be clear but I'm glad :)